apache-mesos-mixin/dashboards/apache-mesos-overview.libsonnet

```diff
@@ -0,0 +1,1383 @@
+            ),
+          ],
+          if $._config.enableLokiLogs then [
+            template.datasource(
+              lokiDatasourceName,
+              'loki',
+              null,
+              label='Loki data source',
+              refresh='load'
+            ),
+          ] else [],
+          [
+            template.new(
+              'job',
+              promDatasource,
+              'label_values(mesos_exporter_build_info,job)',
+              label='Job',
+              refresh=2,
+              includeAll=true,
+              multi=true,
+              allValues='.+',
+              sort=1
+            ),
+            template.new(
+              'instance',
+              promDatasource,
+              'label_values(mesos_exporter_build_info{job=~"$job", instance=~"$instance"}, instance)',
+              label='Instance',
+              refresh=2,
+              includeAll=true,
+              multi=true,
+              allValues='.+',
+              sort=1
+            ),
+            template.new(
+              'mesos_cluster',
+              promDatasource,
+              'label_values(mesos_exporter_build_info{job=~"$job", instance=~"$instance"}, mesos_cluster)',
+              label='Mesos cluster',
+              refresh=2,
+              includeAll=true,
+              multi=true,
+              allValues='.+',
+              sort=1
+            ),
+          ],
+        ])
+      )
+      .addPanels(
+        std.flattenArrays([
+          [
+            masterUptimePanel { gridPos: { h: 6, w: 4, x: 0, y: 0 } },
+            cpusAvailablePanel { gridPos: { h: 6, w: 5, x: 4, y: 0 } },
+            memoryAvailablePanel { gridPos: { h: 6, w: 5, x: 9, y: 0 } },
+            gpusAvailablePanel { gridPos: { h: 6, w: 5, x: 14, y: 0 } },
+            diskAvailablePanel { gridPos: { h: 6, w: 5, x: 19, y: 0 } },
+            memoryUtilizationPanel { gridPos: { h: 6, w: 12, x: 0, y: 6 } },
+            diskUtilizationPanel { gridPos: { h: 6, w: 12, x: 12, y: 6 } },
+            eventsInQueuePanel { gridPos: { h: 6, w: 12, x: 0, y: 12 } },
+            messagesPanel { gridPos: { h: 6, w: 12, x: 12, y: 12 } },
+            registrarStatePanel { gridPos: { h: 6, w: 18, x: 0, y: 18 } },
+            registrarLogRecoveredPanel { gridPos: { h: 6, w: 6, x: 18, y: 18 } },
+            allocatorRow { gridPos: { h: 1, w: 24, x: 0, y: 24 } },
+            allocationRunsPanel { gridPos: { h: 6, w: 6, x: 0, y: 25 } },
+            allocationDurationPanel { gridPos: { h: 6, w: 6, x: 6, y: 25 } },
+            allocationLatencyPanel { gridPos: { h: 6, w: 6, x: 12, y: 25 } },
+            eventQueueDispatchesPanel { gridPos: { h: 6, w: 6, x: 18, y: 25 } },
+            agentsRow { gridPos: { h: 1, w: 24, x: 0, y: 31 } },
+            memoryUtilizationPanel { gridPos: { h: 6, w: 12, x: 0, y: 32 } },
+            diskUtilizationPanel { gridPos: { h: 6, w: 12, x: 12, y: 32 } },
+            logsRow { gridPos: { h: 1, w: 24, x: 0, y: 38 } },
```

Comment:
Shouldn't this be rendered conditionally with  ` $._config.enableLokiLogs`?

Reply:
good catch. Added another loki if condition